python/import_util.py

```diff
@@ -0,0 +1,120 @@
+from datetime import datetime, date, time, timedelta
+import json as js
+from mage.export_import_util.parameters import Parameter
+import mgp
+
+
+def convert_from_isoformat(property):
+    if isinstance(property, str):
+        if str.startswith(property, Parameter.DURATION.value):
+            parsed_time = datetime.strptime(
+                property.split("(")[-1].split(")")[0], "%H:%M:%S.%f"
+            )
+            return timedelta(
+                hours=parsed_time.hour,
+                minutes=parsed_time.minute,
+                seconds=parsed_time.second,
+                microseconds=parsed_time.microsecond,
+            )
+        if str.startswith(property, Parameter.LOCALTIME.value):
+            return time.fromisoformat(property.split("(")[-1].split(")")[0])
+        if str.startswith(property, Parameter.LOCALDATETIME.value):
+            return datetime.fromisoformat(property.split("(")[-1].split(")")[0])
+        if str.startswith(property, Parameter.DATE.value):
+            return date.fromisoformat(property.split("(")[-1].split(")")[0])
+
+    return property
```

Comment:
I have a suggestion to style this differently, since you are actually doing some logic on an indent of 3 or more, it makes sense to first return property if it is not a type of string and then do other things if it is a string.
```suggestion
    if not isinstance(property, str):
      return property
      
    if str.startswith(property, Parameter.DURATION.value):
            parsed_time = datetime.strptime(
                property.split("(")[-1].split(")")[0], "%H:%M:%S.%f"
            )
            return timedelta(
                hours=parsed_time.hour,
                minutes=parsed_time.minute,
                seconds=parsed_time.second,
                microseconds=parsed_time.microsecond,
            )
      if str.startswith(property, Parameter.LOCALTIME.value):
          return time.fromisoformat(property.split("(")[-1].split(")")[0])
      if str.startswith(property, Parameter.LOCALDATETIME.value):
          return datetime.fromisoformat(property.split("(")[-1].split(")")[0])
      if str.startswith(property, Parameter.DATE.value):
          return date.fromisoformat(property.split("(")[-1].split(")")[0])

```

Reply:
It is complicated to style this with Github actions but you see a point I hope 😄

Reply:
Also, I think you can isolate this part in variable:
`smth = property.split("(")[-1].split(")")[0]`

Reply:
Yep, totally makes sense. I added this part with string check later, so I missed this. Thank you! 
Also, isolation of `property.split("(")[-1].split(")")[0]` is great too. Thanks!

Reply:
Is the point of `property.split("(")[-1].split(")")[0]` isolation just readability?

Reply:
Yes, just for readability 😄

Reply:
I changed those two, so feel free to check it and let me know if I did it okay :D